app/client/lib/helpers/helperFunctions.js

```diff
@@ -35,6 +35,25 @@ Helpers.sortByBalance = function(a, b){
     return !b.disabled && new BigNumber(b.balance, 10).gt(new BigNumber(a.balance, 10)) ? 1 : -1;
 };
 
+
+/**
+Return an account you own, from a list of accounts
+
+@method getAccountYouOwn
+**/
+Helpers.getAccountYouOwn = function(accountList){
+    // Load the accounts owned by user and sort by balance
+    var accounts = EthAccounts.find({name: {$exists: true}}, {sort: {name: 1}}).fetch();
```

Comment:
why you are using this `{name: {$exists: true}}`?
Simply `EthAccounts.find({}, {sort: {balance: 1}}).fetch();` should be enough